config/settings.yml

```diff
@@ -572,6 +572,9 @@ github_stats:
   username: github-stats-rake
   token: fake_token
 
+google_analytics:
```

Comment:
settings.yml has a reference to `google_analytics_tracking_id` on line 457.  That should probably move in with this group, it's only used in a few places.

Reply:
just occured to me that would require a corresponding change to devops.
hopefully the keys could be side-by-side if changing devops is too complicated

Reply:
thanks for the suggestion i made a new issue for it https://github.com/department-of-veterans-affairs/va.gov-team/issues/14017